internal/api/installation.go

```diff
@@ -71,6 +71,12 @@ func handleCreateInstallation(c *Context, w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	if len(createInstallationRequest.DNS) >= 64 {
+		c.Logger.Warningf("DNS name too long DNS=%s", createInstallationRequest.DNS)
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
```

Comment:
Cloud we move this check to https://github.com/mattermost/mattermost-cloud/blob/master/model/installation_request.go#L46 ?

We already validate a bunch of stuff when the request is read in and this fits perfectly there.

Also, it would be nice if the error message was something like `DNS names must be less than 64 characters, but name was %d long. DNS=%s`

Reply:
ahh cool, yes will move there, thanks for the review

Reply:
done